src/assets/styles/components/input-range.scss

```diff
@@ -0,0 +1,70 @@
+input[type='range'] {
+    display: block;
+    width: 175px;
+    -webkit-appearance: none;
+    cursor: pointer;
+
+    &:focus {
+        outline: none;
+
+        &::-webkit-slider-runnable-track {
+            outline: none;
+        }
+    }
+
+    &::-webkit-slider-runnable-track,
+    &::-webkit-slider-thumb {
+        -webkit-appearance: none;
+    }
+
+    &::-webkit-slider-thumb {
+        background-color: #ffffff;
+        width: 17px;
+        height: 17px;
+        border: solid 2px #46c0ae;
+        border-radius: 50%;
+        margin-top: -6px;
+    }
+
+    &::-moz-range-thumb {
+        background-color: #ffffff;
+        width: 17px;
+        height: 17px;
+        border: solid 2px #46c0ae;
+        border-radius: 50%;
+    }
+
+    &::-ms-thumb {
+        background-color: #ffffff;
+        width: 17px;
+        height: 17px;
+        border: solid 2px #46c0ae;
+        border-radius: 50%;
```

Comment:
Maybe for future PR's we should add a partial with colors  variables for avoiding repeating across the file.

Reply:
I agree, let's talk to Sergio and we will do it as a refactor for future Hitos

Reply:
how much effort it takes in hours? if it is less than 1 hour please do it and discount time from the next milestone (hito 😬 )